### DISCUSSION TITLE
color white no working

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#10974] [Help] [answered]
className="border-white bg-white outline-white" every color white not working. it's a bug or this bug ocours just to me ?

Look the examples in your site:
https://user-images.githubusercontent.com/93288871/230828800-896300c2-a7d3-4889-8045-b52953825d1a.png
https://user-images.githubusercontent.com/93288871/230829324-ce6f8091-03c4-49ed-89d1-57400df0419e.png

Dont make sense.

**Comment by wongjn** [comment]
Perhaps check whether you have completely overridden the color palette in Tailwind and thus unsetting `white` as a color value and thus unsetting `border-white`, `bg-white`, `outline-white` etc. as a valid class candidates.

**Reply by ghost** [reply]
/** @type {import('tailwindcss').Config} */
export default {
	content: [
		"./index.html",
		"./src/**/*.{ts,tsx}",
	],
	theme: {
		extend: {},
	},
	plugins: [],
};

I just installed tailwind using vite and didn't change anything about the color palette.

**Reply by wongjn** [reply]
Consider checking that the `content` file globs sufficiently cover files that have Tailwind class names. Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by ghost** [reply]
https://stackblitz.com/edit/nextjs-9m3byc?file=pages/index.js

Look this example in nextjs. bg-white is gray border-white is gray and border-black is white. WHYYYYYYYYY

I installed tailwind by next guide and ran it. I don't understand why white doesn't work.
https://tailwindcss.com/docs/guides/nextjs

**Reply by wongjn** [reply]
It seems you have used `border-inherit` which applies `border-color: inherit` which would make the border color gray. If you replace `border-inherit` with a different color, such as `border-black` that you mention, it would color the border black as in this fork: https://stackblitz.com/edit/nextjs-p9t9c7?file=pages%2Findex.js

**Reply by ghost** [reply, marked as answer]
Bro, was my plugin darkmode in chrome. AHHHHHHHHHHHHHHHHHHHHHHHHHHH

Tanks for try. Have a good day!
